crates/driver/src/infra/solver/dto/notification.rs

```diff
@@ -57,7 +57,7 @@ impl Notification {
                         transaction: hash.0,
                     },
                     notify::Settlement::SimulationRevert => Kind::Cancelled,
-                    notify::Settlement::Fail => Kind::Fail,
+                    notify::Settlement::Fail | notify::Settlement::Expired => Kind::Fail,
```

Comment:
But we still send the same type to solvers so they still won't see the difference right? Is this intended?

Reply:
Indeed, unless we just want to have it for logging purposes.

Reply:
This is the legacy solver (singular) crate, and not what we are sending to solvers today.

Reply:
But the introduced `notify::Settlement::Expired` is used here only. Did I miss something?
A bit confused with the following:
> This PR creates an explicit notification subtype for it so that solvers can self-diagnose this case.

Reply:
Oh, I didn't look correctly. This is still part of the driver's crate. Will update